### PR TITLE
[Documentation]: Add CONTRIBUTING.md to ease and explain contribution guidelines.

### PR DESCRIPTION
**Pull request by pathange-s**
… guidelines

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#### Description
+
+- A short description of the changes you made in the commit.
+
+#### Example
+
+- `[Feature]: Implement Login Flow`
+- `[Documentation]: Add Code of Conduct for neutralinojs-cli`
+- `[Tests]: Write Integration tets for submodules of neutralinojs`
```
Comment (shalithasuranga) [inline CONTRIBUTING.md:48]: Hey.. Thanks for sending this great documentation. I have a small request for this section. Please note that we don't follow (or ask contributors to) follow a strict commit message format. You can read the reason behind it from : https://codezri.org/docs/releasezri/intro
 So, could you please submit the markdown file without this section.

Thanks :)

Comment (shalithasuranga) [inline CONTRIBUTING.md:48]: Hey.. Thanks for sending this great documentation. I have a small request for this section. Please note that we don't follow (or ask contributors to) follow a strict commit message format. You can read the reason behind it from : https://codezri.org/docs/releasezri/intro
 So, could you please submit the markdown file without this section.

Thanks :)

**Comment by pathange-s** [comment]
Ah, didnt look into that before. Great then ! Updating it.

**Review by shalithasuranga — APPROVED** [review]
Looks great.. Thank you very much :+1:

**Pull request merged**